src/rgw/rgw_rest_swift.cc

```diff
@@ -2565,7 +2565,8 @@ bool RGWSwiftWebsiteHandler::is_web_dir() const
   /* First, get attrset of the object we'll try to retrieve. */
   RGWObjectCtx& obj_ctx = *static_cast<RGWObjectCtx *>(s->obj_ctx);
   obj.set_atomic(&obj_ctx);
-  obj.set_prefetch_data(&obj_ctx);
+  prefetch_range prefetch{0, static_cast<uint64_t>(s->cct->_conf->rgw_max_chunk_size), true};
+  obj.set_prefetch_data(&obj_ctx, prefetch);
```

Comment:
it looks like `RGWSwiftWebsiteHandler::is_web_dir()` and `RGWSwiftWebsiteHandler::is_index_present()` are only testing for existence, right? we shouldn't prefetch data for either